composeml/label_maker.py

```diff
@@ -172,19 +202,19 @@ def iloc(index, i):
                     if df_slice.index.size > 1 and is_overlap.any():
                         df_slice = df_slice[~is_overlap]
 
-            metadata['window'] = (cutoff_time, window_end)
+            df_slice.context.window = (cutoff_time, window_end)
 
             if isinstance(gap, int):
                 gap_end = iloc(df.index, gap)
-                metadata['gap'] = (cutoff_time, gap_end)
+                df_slice.context.gap = (cutoff_time, gap_end)
```

Comment:
is `cutoff_time` the right argument here? shouldn't the start of the gap be the end of the pervious slices window?

Reply:
The start of the gap is based off the cutoff time to allow for rolling windows.